lib/kitchen/transport/winrm.rb

```diff
@@ -122,9 +133,9 @@ def upload(locals, remote)
         def wait_until_ready
           delay = 3
           session(
-            :retries  => max_wait_until_ready / delay,
-            :delay    => delay,
-            :message  => "Waiting for WinRM service on #{endpoint}, " \
+            retries: max_wait_until_ready / delay,
```

Comment:
I can flip the syntax back, but due to the aruba update, we can only test on 2.x and newer.